### ISSUE TITLE
What Perl dependencies does this product require?

### ISSUE THREAD

**Issue by nvie** [miyagawa/github-growler#2]
Maybe explain how to install Perl dependencies? Or maybe even include them in your product?

I get the following errors in Console.app:

```
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]   dyld: lazy symbol binding failed: Symbol not found: _Perl_Tstack_sp_ptr
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]     Referenced from: /Applications/Github Growler.app/Contents/Resources/extlib/lib/perl5/darwin-thread-multi-2level/auto/XML/LibXML/LibXML.bundle
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]     Expected in: dynamic lookup
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]   dyld: Symbol not found: _Perl_Tstack_sp_ptr
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]     Referenced from: /Applications/Github Growler.app/Contents/Resources/extlib/lib/perl5/darwin-thread-multi-2level/auto/XML/LibXML/LibXML.bundle
01-06-10 16:44:09   [0x0-0x1331330].com.example.miyagawa.GithubGrowler[98208]     Expected in: dynamic lookup
```

**Comment by idris** [comment]
I'm having the same issue. Hope this gets fixed soon!

**Comment by miyagawa** [comment]
The binary is only supported in Leopard because Snow Leopard uses 64bit and perl's Growl binding doesn't work under the 64bit mode. All the perl dependencies are defined in Makefile.PL so you can install by yourself, using tools like cpanminus.

**Comment by idris** [comment]
If anyone's interested in a Github growler for Snow Leopard, you can try my new project: http://github.com/idris/growlhub

**Comment by miyagawa** [comment]
This is now fixed in v2.0.0 :)
